packages/sdk/src/cosmwasmclient.ts

```diff
@@ -2,14 +2,24 @@ import { Sha256 } from "@iov/crypto";
 import { Encoding } from "@iov/encoding";
 
 import { Log, parseLogs } from "./logs";
-import { BlockResponse, BroadcastMode, RestClient, TxsResponse } from "./restclient";
-import { CosmosSdkAccount, CosmosSdkTx, StdTx } from "./types";
+import { BroadcastMode, RestClient } from "./restclient";
+import { Coin, CosmosSdkTx, StdTx } from "./types";
 
 export interface GetNonceResult {
   readonly accountNumber: number;
   readonly sequence: number;
 }
 
+export interface Account {
+  /** Bech32 account address */
+  readonly address: string;
+  readonly balance: ReadonlyArray<Coin>;
+  /** Bech32 encoded pubkey */
```

Comment:
I find the pubkey object (algorithm, data) format easier to handle outside of this package. Unless we decide bech32 pubkey is the only public interface everywhere (like from the Pen), I would prefer to use the object version for consistency, and just use the bech32 internally.

Reply:
eg, if I want to check the algorithm type... do I want to grab a few bytes and check for the amino-encoding in some file? Would anyone but us two even get that idea?

Reply:
Good point